test/python/simulator/grover.py

```diff
@@ -1,8 +1,10 @@
-from qiskit.algorithms import Grover
+from qiskit import BasicAer
 from qiskit.algorithms import AmplificationProblem
+from qiskit.algorithms import Grover
 from qiskit.circuit.library import PhaseOracle
-from qiskit import BasicAer
+
 from mqt.ddsim import DDSIMProvider
```

Comment:
At some point in the (hopefully near) future, it would really be great to introduce pre-commit and all the Python checks that come with it like `mypy` and `isort` so that things like these are automatically handled in a consistent fashion.